.travis.yml

```diff
@@ -1,22 +1,21 @@
 osx_image: xcode10.1
 jobs:
   include:
-    - stage: test Linux
+    - stage: Run tests
       script: ./Scripts/travis/test-Linux.sh
       os: linux
       dist: xenial
-    - stage: test macOS
+    - os: osx
```

Comment:
I'm not up on how travis yml works. Is running the osx tests now part of the "Run tests" stage?

Reply:
Yes the Travis syntax is really opaque sometimes, but this essentially runs the Linux test and osx tests in parallel instead of in sequence.

Reply:
And yes both jobs are part of the "run tests" stage